samples/bookinfo/src/build-services.sh

```diff
@@ -26,48 +26,75 @@ VERSION=$1
 PREFIX=$2
 SCRIPTDIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 
+# Docker build variables
+ENABLE_MULTIARCH_IMAGES=${ENABLE_MULTIARCH_IMAGES:-"false"}
+
+if [ "${ENABLE_MULTIARCH_IMAGES}" == "true" ]; then
+  PLATFORMS="linux/arm64,linux/amd64"	
+  DOCKER_BUILD_ARGS="docker buildx build --platform ${PLATFORMS} --push"
+  # Install QEMU emulators
+  docker run --rm --privileged tonistiigi/binfmt --install all
```

Comment:
nit: can we use `docker run --rm --privileged gcr.io/istio-testing/qemu-user-static --reset -p yes`?

Reply:
we found some build flakiness in envoy builds that was attributed to the qemu libs used in `multiarch/qemu-user-static`
https://github.com/multiarch/qemu-user-static/issues/131, its probably fixed, but `tonistiigi/binfmt` tends to find and fix such issues faster